pkg/storaged/stratis-details.jsx

```diff
@@ -726,13 +937,28 @@ const StratisStoppedPoolSidebar = ({ client, uuid }) => {
 };
 
 export const StratisStoppedPoolDetails = ({ client, uuid }) => {
+    const stopped_props = client.stratis_manager.StoppedPools[uuid];
+    const keydesc_info = stopped_props.key_description;
+    const clevis_info = stopped_props.clevis_info;
+
+    const encrypted = (keydesc_info && keydesc_info.v[0]) || (clevis_info && clevis_info.v[0]);
+    const keydesc = keydesc_info && keydesc_info.v[0] && keydesc_info.v[1].v[0] && keydesc_info.v[1].v[1];
+    const can_tang = encrypted && clevis_info && clevis_info.v[0] && (!clevis_info.v[1].v[0] || clevis_info.v[1].v[1][0] == "tang");
+    const tang_url = can_tang && clevis_info.v[1].v[0] ? JSON.parse(clevis_info.v[1].v[1][1]).url : null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230602-125216-41c010ea-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)